src/tests/compiler/usecheck.rs

```diff
@@ -166,9 +166,9 @@ fn verify_use_check_with_singleton_top_layer_works() {
     let res = do_basic_run(&vec![
         "run".to_string(),
         "-i".to_string(),
-        "resources/tests".to_string(),
-        "-i".to_string(),
         "resources/tests/usecheck-work".to_string(),
+        "-i".to_string(),
+        "resources/tests".to_string(),
```

Comment:
I don't understand the relevance of this change. is the order important here?

Reply:
Yes, this is the include paths and this change adds a canonical version of curry-and-treehash.clinc in the main directory. 
 I've been thinking about this and probably will remove the other ad-hoc version in usecheck-work, and just let usecheck-fail supercede this directory.  I'll likely add more example code that relies on the file from chia-blockchain over time anyway so the correct version doesn't really "belong" to the use checker anymore.

Reply:
What i decided to do is consolidate so that the canonical version of this include is in the resources/tests dir and usecheck-fail will supercede it when desired.